src/main/java/seedu/duke/logic/parser/exceptions/InvalidPositionException.java

```diff
@@ -0,0 +1,7 @@
+package seedu.duke.logic.parser.exceptions;
+
+public class InvalidPositionException extends ParserException {
+    public InvalidPositionException() {
+        super("Invalid position chosen! Make sure the single letter position used is shown within the tree seen.");
```

Comment:
```suggestion
        super("Invalid position chosen! Make sure the single letter position specified is depicted in the current logic diagram.");
```
Cos we don't want them to be confused by calling it a tree?